DOKS-automatic-node-repair/README.md

```diff
@@ -1 +1,374 @@
+    ```
+3. In order to interact with the cluster from the CLI `doctl` has to be installed. For example, use the following commands on MacOS:
+    
+    ```bash
+    brew info doctl
+    brew install doctl
+    ```
+4. If `doctl` authentication hasn't been set up yet please do it now via:
+   
+    ```bash
+    doctl auth init --access-token "$DIGITAL_OCEAN_TOKEN"
+    ```
+    The output looks similar to the following (token value is masked):
+    ```
+    Using token [*****************************************]
+
+    Validating token... OK
+    ```
+5. Spin up the cluster and wait for it to be provisioned:
+
+    ```bash
+    export CLUSTER_NAME="mobius-testing-cluster"  # change as required
+    export CLUSTER_REGION="lon1"                  # grab a region that's more close to you from: doctl k8s options regions
+    export CLUSTER_NODE_SIZE="s-2vcpu-4gb"        # change as required
+    export CLUSTER_NODE_COUNT=2                   # need 2 nodes at least to test the scenarios
+    export CLUSTER_NODE_POOL_NAME="mbt-np"        # change as required
+    export CLUSTER_NODE_POOL_TAG="mbt-cluster"    # change as required
+    export CLUSTER_NODE_POOL_LABEL="type=basic"   # change as required
+
+    doctl k8s cluster create "$CLUSTER_NAME" \
+      --auto-upgrade=false \
+      --node-pool "name=${CLUSTER_NODE_POOL_NAME};size=${CLUSTER_NODE_SIZE};count=${CLUSTER_NODE_COUNT};tag=${CLUSTER_NODE_POOL_TAG};label=${CLUSTER_NODE_POOL_LABEL}" \
+      --region "$CLUSTER_REGION"
+    ```
+
+    The output looks similar to the following:
+    ```
+    Notice: Cluster is provisioning, waiting for cluster to be running
+    ......................................................................
+    Notice: Cluster created, fetching credentials
+    Notice: Adding cluster credentials to kubeconfig file found in "~/.kube/config"
+    Notice: Setting current-context to do-lon1-mobius-testing-cluster
+    ID                                      Name                       Region    Version        Auto Upgrade    Status     Node Pools
+    11bdd0f1-8bd0-42dc-a3af-7a83bc319295    mobius-testing-cluster     lon1      1.21.2-do.2    false           running    basicnp
+    ```
+
+6. Get worker nodes status:
+
+    ```bash
+    kubectl get nodes
+    ```
+
+    The output looks similar to the following:
+    ```
+    NAME            STATUS   ROLES    AGE     VERSION
+    basicnp-8k4ep   Ready    <none>   2m52s   v1.21.2
+    basicnp-8k4es   Ready    <none>   2m16s   v1.21.2
+    ```
+    ### Note:
+
+      If for some reason the `kubectl` context was not set properly we can always fix it via:
+      
+      ```bash
+      doctl k8s cluster kubeconfig save "$CLUSTER_NAME"
+      ```
+
+## Digital Mobius
+
+### Overview
+
+[Digital Mobius](https://github.com/Qovery/digital-mobius) is an open source project written in the `GO` language which can automatically recycle unhealthy nodes from a `DigitalOcean Kubernetes` cluster (DOKS).
```

Comment:
nit: should be"Go", not "GO".

Reply:
Thanks for the suggestion. Fixed in the latest commit.